docs/_includes/common/signal.html

```diff
@@ -0,0 +1,61 @@
+<div id="signal">
+    {% include common/section-header.html name="signal" version=page.versions.signal %}
+
+    <p>Signals are data-backed recommendations to help customers make more informed decisions.</p>
+    <p>There are four signal types, each with its own corresponding modifier class: trustworthy, recent, time-sensitive &amp; neutral.</p>
+
+    <h3 id="span-signal">A Signal made with a span.</h3>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="demo__signal">
+                <span class="signal signal--trustworthy">
+                    Trustworthy
+                </span>
+                <span class="signal signal--recent">
+                    Recent
+                </span>
+                <span class="signal signal--time-sensitive">
+                    Time Sensitive
+                </span>
+                <span class="signal signal--neutral">
+                    Neutral
+                </span>
+            </div>
+        </div>
+    </div>
+    {% highlight html %}
+<span class="signal signal--trustworthy">Trustworthy</span>
+<span class="signal signal--recent">Recent</span>
+<span class="signal signal--time-sensitive">Time Sensitive</span>
+<span class="signal signal--neutral">Neutral</span>
+    {% endhighlight %}
+
+    <h3 id="div-signal">A Signal can be made with a div as well as a span.</h3>
```

Comment:
I don't think we need this div example anymore (now that it's always inline-block, regardless). Also you can delete the h3 above and the demo__signal div.